ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -48,13 +51,20 @@ open class AvatarData: NSObject, Avatar {
 
     @objc public var presence: Presence
 
+    @objc public var showsBorder: Bool
+
     @objc public var hideInsideGapForBorder: Bool = false
 
-    @objc public init(primaryText: String = "", secondaryText: String = "", image: UIImage? = nil, presence: Presence = .none, color: UIColor? = nil) {
+    @objc public init(primaryText: String = "", secondaryText: String = "", image: UIImage? = nil, presence: Presence = .none, color: UIColor? = nil, showsBorder: Bool = false) {
         self.primaryText = primaryText
         self.secondaryText = secondaryText
         self.image = image
         self.presence = presence
         self.color = color
+        self.showsBorder = showsBorder
+    }
+
+    @objc public convenience init(primaryText: String = "", secondaryText: String = "", image: UIImage? = nil, presence: Presence = .none, color: UIColor? = nil) {
```

Comment:
Is this for devmain, compatibility?

Reply:
This change will be backported to the v0.1 branch, so yep, compatibility is critical.

As for the enum you mentioned, that only applies to the `AvatarGroup` itself. Individual `Avatar` continues to use a `Boolean` for their state, since an individual one can only be on or off. But again for compatibility, I set the default in `AvatarGroup` to `.none`, which was the historical default.